gio/sys/Cargo.toml

```diff
@@ -89,4 +89,4 @@ version = "2.68"
 version = "2.70"
 
 [package.metadata.system-deps.gio_2_0.v2_72]
-version = "2.71"
```

Comment:
is this change actually needed?

Reply:
In what sense? The API of various things added between 2.70 and 2.72 changed a couple of times during the 2.71 development cycle, so we can only really guarantee that it works with 2.72.0 for sure.